examples/iterative_sense_reconstruction.py

```diff
@@ -0,0 +1,140 @@
+# %% [markdown]
+# # Iterative SENSE Reconstruction of 2D golden angle radial data
+# Here we use the IterativeSenseReconstruction class to reconstruct images from ISMRMRD 2D radial data
+# %%
+# define zenodo URL of the example ismrmd data
+zenodo_url = 'https://zenodo.org/records/10854057/files/'
+fname = 'pulseq_radial_2D_402spokes_golden_angle_with_traj.h5'
+# %%
+# Download raw data
+import tempfile
+from pathlib import Path
+
+import requests
+
+data_folder = Path(tempfile.mkdtemp())
+data_file = tempfile.NamedTemporaryFile(dir=data_folder, mode='wb', delete=False, suffix='.h5')
```

Comment:
why do we need data_folder = Path(tempfile.mkdtemp()) ?
tempfile.NamedTemporaryFile wihout the optional dir already uses the temp dir..

Reply:
In order to delete the folder at the end of the script (this was missing). Alternative would be to use something like shutil.rmtree(os.path.dirname(data_folder)) or simply just delete the file and leave the empty folder. Any preferences?

Reply:
just don't create the folder!

Reply:
and to delete it it would be `data_file.delete`

Reply:
I hadn't realised that tempfile folders and files behave differently and that files are deleted automatically whereas folders most of the time are not (except when used in a context)